test/features/support/env.js

```diff
@@ -0,0 +1,16 @@
+// Environment global variables declaration
```

Comment:
code style needs to align with standardJS also use const/let instead of var accordingly